src/scripts/main.js

```diff
@@ -1,3 +1,33 @@
 'use strict';
 
-// write your code here
+function formatNumber(num) {
+  return num.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',');
+}
+
+const populationElements = document.querySelectorAll('.population');
```

Comment:
The `querySelectorAll` method is used correctly to select elements with the class 'population'. Ensure that these elements exist in the HTML to avoid errors.